app/src/main/res/values/strings.xml

```diff
@@ -797,8 +797,8 @@ How to translate with transifex:
     <string name="nc_conversation_not_found">Conversation not found</string>
     <string name="add_to_notes">Add to Notes</string>
     <string name="nc_edited_by_admin">Edited by admin</string>
-    <string name="nc_edited_by">"Edited by "</string>
-    <string name="nc_clear_edit_button">clear Edit Button</string>
+    <string name="nc_edited_by">"Edited by"</string>
```

Comment:
@sowjanyakch 
seeing this now, i suggest to change how this is shown, because for some languages the "Edited by" part might make more sense after the username.

Could you change it to

<string name="nc_edited_by">Edited by %1$s</string>

and then when using this key in code you can use String.format like this:

String.format(
            context.getString(R.string.nc_edited_by),
            message.lastEditActorDisplayName
        )

With this solution the whitespace is also better handled as translators won't think "oh a whitespace at the end of a string, let's delete it"

Reply:
Agreed. I will make the changes.